home/compile_windows.md

```diff
@@ -49,6 +49,32 @@ You can also [compile with meson](https://github.com/revbayes/revbayes/blob/deve
     **Boost and CMake:**
 
     It's important that the version of Boost that you use be supported by the version of CMake that you use. You can check this by going to the package source for the CMake version you're using e.g. [3.14.5](https://github.com/Kitware/CMake/blob/v3.14.5/Modules/FindBoost.cmake). Search for `_Boost_KNOWN_VERSIONS` and ensure your boost version appears in the list.
+    
+    Occasionally, RevBayes may require a version of cmake or Boost that is not available via Cygwin.
+    Once you have used Cygwin to install gcc, you can install these separately.
+    
+    [cmake](https://cmake.org/download) has a simple and well-documented installation pathway.
+    
+    Configuring [Boost](https://www.boost.org/) is less straightforward.
+    
+    1. [Download Boost](https://www.boost.org/users/download/) and unzip the archive.  In this tutorial, I unzipped v1.82.0 to `c:\boost\boost_1_82_0`.  It may be wise not to use the most current version; at time of writing, the latest 1.85 release caused compatibility issues.
+    2. Use Open a cygwin terminal window and `cd` to the boost directory, here: `c:/boost/boost_1_82_0`
+    3. Type `./bootstrap.bat gcc` to run the script with the gcc toolset.
+    4. Execute `b2 toolset=gcc-13 address-model=64 architecture=x86 --build-dir=build variant=release --build-type=complete --prefix=c:/boost/boost_1_82_0/gcc --with-regex --with-program_options --with-thread --with-system --with-filesystem --with-date_time --with-serialization  install`.
+       - The `address-model=64 architecture=x86 toolset=gcc-13` options are assumed to match the configuration of your system – here, a 64-bit x86 machine – throughout this tutorial.  Use `gcc -v` to check which version of gcc you are using and update `gcc-13` to match.
+    5. Check that installation was successful.  If it was, `C:\boost\boost_1_82_0\gcc\lib\cmake\Boost-1.82.0` will contain a file `BoostConfig.cmake`
+    
+    If you see errors, you may need to use a more recent version of b2 than is bundled with Boost:
+    - Download [b2 5.2.0](https://github.com/bfgroup/b2/releases) or above.
+    - Remove (or rename) the local copy of `b2.exe`, so the system uses the updated version installed globally above.
+    - Run `b2 --version` to confirm that a version of b2 >= 5.2 is available.
+    
+    6. Update `<path/to/revbayes>/src/CMakeLists.txt` to point to Boost:
+      a. After the line `project(RevBayes)`, add:
+         `set(BOOST_ROOT "C:/boost/boost_1_82_0/gcc")` 
+         (to match the value specified to b2 in the `prefix` argument)
+      b. After the  line starting `find_package(Boost`, around line 170,
```

Comment:
I believe this is now addressed by 
https://github.com/revbayes/revbayes/pull/498